Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -590,10 +590,9 @@ String InspectorClient::generate_dom_tree(JsonObject const& dom_tree)
         append_data_attribute("id"sv, node_id);
 
         if (type == "text"sv) {
-            auto deprecated_text = node.get_byte_string("text"sv).release_value();
-            deprecated_text = escape_html_entities(deprecated_text);
+            auto deprecated_text = node.get_string("text"sv).release_value();
 
-            auto text = MUST(Web::Infra::strip_and_collapse_whitespace(deprecated_text));
+            auto text = MUST(Web::Infra::strip_and_collapse_whitespace(escape_html_entities(deprecated_text)));
```

Comment:
The extra `escape_html_entities()` calls should be in a separate commit.

Reply:
These are not extra calls, they are moved into expressions because the variables they were assigned to changed types.